apps/condo/domains/marketplace/schema/Invoice.test.js

```diff
@@ -598,6 +599,7 @@ describe('Invoice', () => {
                 code: 'BAD_USER_INPUT',
                 type: 'INVOICE_ALREADY_PAID',
                 message: 'Changing of paid invoice is forbidden',
+                messageForUser: 'marketplace.invoice.error.alreadyPaid',
```

Comment:
usually in translations of server errors the prefix `api` is used.
`api.marketplace.marketplace.invoice.error.alreadyPaid`

Reply:
ok, renamed